src/components/MediaSettings/VideoBackgroundEditor.vue

```diff
@@ -110,10 +140,111 @@ export default {
 		},
 	},
 
+	async mounted() {
+		// userRoot path
+		const userRoot = '/files/' + this.$store.getters.getUserId()
+
+		// Relative background folder path
+		const relativeBackgroundsFolderPath = this.$store.getters.getAttachmentFolder() + '/Backgrounds'
+
+		// Absolute backgrounds folder path
+		const absoluteBackgroundsFolderPath = userRoot + relativeBackgroundsFolderPath
+
+		// Create the backgrounds folder if it doesn't exist
+		if (await client.exists(absoluteBackgroundsFolderPath) === false) {
+			await client.createDirectory(absoluteBackgroundsFolderPath)
+		}
```

Comment:
**thought:** unhandled exception

These calls could potentially fail. I'm not sure, how we should react in this case, but without handling exception on mounted component probably crashes.